robot/board.py

```diff
@@ -56,7 +63,7 @@ def _connect(self, socket_path):
         try:
             self.socket.connect(str(self.socket_path))
         except ConnectionRefusedError as e:
-            print('Error connecting to:', socket_path)
+            print('Error connecting to: ', self.socket_path)
```

Comment:
I don't think we need the extra space here (though the change to add `self.` on `socket_path` is correct):
``` python
>>> print('a', 42)
a 42
>>> print('a ', 42)
a  42
```

Reply:
hmm, odd, i'd always assumed `print` just did a concatenation behind the scenes. Will adjust